packages/babel-plugin-transform-object-rest-spread/README.md

```diff
@@ -20,6 +20,15 @@ $ npm install babel-plugin-transform-object-rest-spread
 }
 ```
 
+This plugin will polyfill `Object.assign` by default. If you want to prevent that use config this
```

Comment:
Was just saying maybe `If you want to prevent that use config this instead:`
could be something like `If you want to prevent that use a config like this instead`